plans/objchange/compatible.go

```diff
@@ -169,6 +169,14 @@ func assertObjectCompatible(schema *configschema.Block, planned, actual cty.Valu
 			})
 			errs = append(errs, setErrs...)
 
+			if maybeUnknownBlocks {
+				// When unknown blocks are present the final blocks may be
+				// at different indices than the planned blocks, so unfortunately
```

Comment:
The sentiment of this comment is correct for both cases, but in the set case it's not really _indices_ that are the problem... maybe best to generalize this comment so it isn't confusing for a future reader who doesn't realize that this text was originally written for the list codepath.

Reply:
oh yes, thanks. That was obviously a quick copy&paste ;)